1_send_data/README.md

```diff
@@ -0,0 +1,20 @@
+# Send data to LiveObjects
+
+This example show how to send some example data(device uptime) to LiveObjects using Arduino MKR1500

```

Comment:
'This example shows how to send some sample data (device uptime) to Live Objects using Arduino MKR NB 1500.'